pkgs/desktops/gnome-3/extensions/text-translator/fix-deps.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/extension.js b/extension.js
+index 421a2aa..6b84ef9 100644
+--- a/extension.js
++++ b/extension.js
+@@ -6,6 +6,7 @@
+             GIRepository.Repository.prepend_search_path("/usr/lib/" + path);
+         }
+     );
++    GIRepository.Repository.prepend_search_path('@clutter@');
```

Comment:
As mentioned in the comments, `extension.js` does not need path to clutter, it will get it from GNOME Shell.

Reply:
Ping.

Reply:
Sorry, I'm gonna fix it.